### ISSUE TITLE
Brand Icons not included.

### ISSUE THREAD

**Issue by Samistine** [Semantic-Org/Semantic-UI-CSS#49]
Brand icons do not seem to be included in the build. If this is intended behavior, please document it.

**Comment by piotrwawrzyn** [comment]
> [Semantic-UI-CSS/package.js](https://github.com/Semantic-Org/Semantic-UI-CSS/blob/980e4be53fc3b3e45ee1fb09f44b41eaadab4f37/package.js#L18-L32)
> 
> Lines 18 to 32 in [980e4be](/Semantic-Org/Semantic-UI-CSS/commit/980e4be53fc3b3e45ee1fb09f44b41eaadab4f37)
> 
>  api.addFiles([ 
>    // icons 
>    'themes/default/assets/fonts/icons.eot', 
>    'themes/default/assets/fonts/icons.svg', 
>    'themes/default/assets/fonts/icons.ttf', 
>    'themes/default/assets/fonts/icons.woff', 
>    'themes/default/assets/fonts/icons.woff2', 
>   
>    // flags 
>    'themes/default/assets/images/flags.png', 
>   
>    // release 
>    'semantic.css', 
>    'semantic.js' 
>  ], 'client'); 
> 
> should definitely be put in there.
> small work around until it get's fixed for your ember-cli-build.js:
> 
> ```js
> 'eot,svg,ttf,woff,woff2'.split(',').forEach(type => {
>   ['brand', 'outline'].forEach(asset => {
>     app.import(`node_modules/semantic-ui-css/themes/default/assets/fonts/${asset}-icons.${type}`, {
>       destDir: 'assets/themes/default/assets/fonts'
>     });
>   });
> });
> ```

How do I fix this if I'm not using ember-cli?

**Comment by rainrivas** [comment]
Any updates on possibly when the update will be pushed?

**Comment by YassienW** [comment]
Since this is probably not getting fixed in the near future (if ever), does anyone have a workaround?

**Comment by amerryma** [comment]
> Since this is probably not getting fixed in the near future (if ever), does anyone have a workaround?

Could try this: https://fomantic-ui.com/elements/icon.html#brands

**Comment by amerryma** [comment]
> Since this is probably not getting fixed in the near future (if ever), does anyone have a workaround?

Could try this: https://github.com/fomantic/Fomantic-UI-CSS

**Comment by YassienW** [comment]
Nope, using fomantic's CSS doesn't add the icons i need, those icons are missing from [Fomantic's website ](https://fomantic-ui.com/elements/icon.html#brands) too.

**Comment by amerryma** [comment]
> Nope, using fomantic's CSS doesn't add the icons i need, those icons are missing from [Fomantic's website ](https://fomantic-ui.com/elements/icon.html#brands) too.

Which brand icons are missing? This is what I see.
![Screen Shot 2020-03-17 at 11 47 37 AM](https://user-images.githubusercontent.com/6307371/76890732-228e3a80-6845-11ea-934b-93259bb285f9.png)

**Comment by YassienW** [comment]
Facebook, instagram, twitter, linkedin for example are missing

**Comment by amerryma** [comment]
> Facebook, instagram, twitter, linkedin for example are missing

I see them. 
![Screen Shot 2020-03-17 at 11 51 26 AM](https://user-images.githubusercontent.com/6307371/76891104-ae07cb80-6845-11ea-97b6-bda10c933e5c.png)

**Comment by YassienW** [comment]
Hmm i think my cache was messed up, i'll see if i can get it working with Fomantic.

**Comment by YassienW** [comment]
Just in case this happens to someone; the icons are working but they were getting blocked by my ad blocker.
